docs/players/Installation_iOS.md

```diff
@@ -54,7 +53,7 @@ The easiest way to install the ipa on your device is to do one of the following:
 
 Alternatively, to install the signed ipa on your device, you can use Xcode or Apple Configurator (available on the Mac App Store for free). The latter also allows installing ipa from the command line, here's an example that assumes you have only 1 device connected to your Mac and the signed ipa is on your desktop:
 
-    /Applications/Apple\ Configurator.app/Contents/MacOS/cfgutil install-app ~/Desktop/vcmi.ipa
+`/Applications/Apple\ Configurator.app/Contents/MacOS/cfgutil install-app ~/Desktop/vcmi.ipa`
```

Comment:
why is this change needed?

Reply:
Goal was unification - in some places we use 4 spaces, but mostly we use backquotes to format code embeds.

There were also a couple of places where 4 spaces were intended to be used as continuation of a sublist, but were interpreted as code embed

However looks like at least from Github point of view this is not equivalent replacement:

    4 spaces

```
Triple backtick
```

`Single backtick`

So will replace with triple backtick to preserve pre-existing formatting.

Reply:
Yes, 4 spaces are identical to triple backtick, except that you can't specify language name